packages/transactions/src/coders/input.ts

```diff
@@ -57,78 +58,47 @@ export type InputCoin = {
 };
 
 export class InputCoinCoder extends Coder<InputCoin, InputCoin> {
-  constructor() {
-    super('InputCoin', 'struct InputCoin', 0);
-  }
+  private primary = coders.struct({
+    txID: coders.b256,
+    outputIndex: coders.u16,
+    owner: coders.b256,
+    amount: coders.u64,
+    assetId: coders.b256,
+    txPointer: new TxPointerCoder(),
```

Comment:
Why doesn't this use the new [txPointerCoder](https://github.com/FuelLabs/fuels-ts/blob/ps/feat/abi-coder/packages/transactions/src/coders/tx-pointer.ts#L14-L17) ?

Reply:
`txPointerCoder` is not exported and only used by the `TxPointerCoder` class. I've tried to minimise the breaking changes from the transaction package.